pkg/flavor/types.go

```diff
@@ -35,4 +35,5 @@ type Flavor interface {
 	GetNFSVolumeID(volumeID string) (string, error)
 	GetOrchestratorVersion() (*version.Info, error)
 	MonitorPod(podLabelkey, podLabelvalue string) error
+	GetSnapshotNameFromSnapshot(snapshotName string) (string, error)
```

Comment:
This function name is kind of hard to understand.  It seems like it should take a snapshot parameter.  Should it be something like GetGroupSnapshotNameFromSnapshotName?